### ISSUE TITLE
Adopters can confirm their drain is cleaned / log their work

### ISSUE THREAD

**Issue by hackajesse** [sfbrigade/adopt-a-drain#45]
Via SMS or the app please (no email)? :)

**Comment by hackajesse** [comment]
[Oakland's time tracking form](https://docs.google.com/forms/d/1UphXhPsn0BtVsquilDYnZDfcirO7xvt1sUnh-OoCj28/viewform?c=0&w=1)
There is also this [volunteer tracking app](https://www.trackitforward.com/) - note that it's paid

**Comment by jeanwalshie** [comment]
i like this! i know you don't want it via email @hackajesse but the simplest thing could be to create a tracking form like the one above and email them the link periodically/when we send a storm reminder/end of rainy season, etc. 
Maybe the tracking form link could be included in the pop up box for "Adopted by You" drains, above the red "Abandon this Drain" box.

**Comment by jeanwalshie** [comment]
i'd like to revive this discussion. tracking drain cleaning is quite important. 
what if users had the option of logging activity in this window, say above the Abandon this Drain button? Not sure where the info would go...
![image](https://cloud.githubusercontent.com/assets/15933453/13933289/9f1ffefc-ef69-11e5-9597-beebbad0990b.png)

**Comment by jszwedko** [comment]
Agreed, this feature would need a lot of upfront discovery and design.

**Comment by jeanwalshie** [comment]
i'm going to create a new issue as a temporary fix for this.
